source/speech/speech.py

```diff
@@ -160,10 +161,17 @@ def isBlank(text):
 RE_CONVERT_WHITESPACE = re.compile("[\0\r\n]")
 
 
-def processText(locale: str, text: str, symbolLevel: characterProcessing.SymbolLevel) -> str:
+def processText(
+		locale: str,
+		text: str,
+		symbolLevel: characterProcessing.SymbolLevel,
+		normalize: bool = False
+) -> str:

```

Comment:
can you add a docstring, particularly for the behaviour of normalise

Reply:
Sure, will do